Generate/general/xslt/generateTestScript.xsl

```diff
@@ -422,6 +406,28 @@
         <xsl:value-of select="$fullFilename"/>
     </xsl:function>
     
+    
+    <!-- Construct a file path from the elements.
+         param base is the folder where the file resides
+         param filename is the name of the file in the folder including the extension

```

Comment:
Does not specifically check if the extension exists though...

Reply:
Dat gebeurt op het moment dat je hem resultaat probeert te lezen.